alarm_control_panel-card/alarm_control_panel-card.js

```diff
@@ -29,14 +14,47 @@ class AlarmControlPanelCard extends HTMLElement {
   }
 
   set hass(hass) {
-    this.myhass = hass;
     const entity = hass.states[this._config.entity];
-    if (entity && entity.state != this._state) {
-      this._state = entity.state;
-      this._updateCardContent(entity);
+
+    if (entity) {
+      this.myhass = hass;
+      if(!this.shadowRoot.lastChild) {
+        this._createCard(entity);
+      }
+      if (entity.state != this._state) {
+        this._state = entity.state;
+        this._updateCardContent(entity);
+      }
     }
   }
 
+  _createCard(entity) {
+    const config = this._config;
+
+    const card = document.createElement('ha-card');
+    const content = document.createElement('div');
+    content.id = "content";
+    content.innerHTML = `
+      ${config.title ? '<div id="state-text"></div>' : ''}
+      <ha-icon id="state-icon"></ha-icon>
+      ${this._actionButtons()}
+      ${entity.attributes.code_format ?
+          `<paper-input label='${this._label("ui.card.alarm_control_panel.code")}'
```

Comment:
Are translations available on the hass object to make sense to use such complex ids?

Reply:
I don't understand this question. Translations seem to be loaded and associated with the hass object by default (I didn't write anything, they were there). The ids are complex but that's what is there.